cve_bin_tool/fetch_json_db.py

```diff
@@ -74,6 +88,13 @@ def get_download_urls(self, session):
                     for year in self.metadata["db"][key]
                 ]
             )
+            if self.metadata["signed"]:
```

Comment:
This one probably has to be rethought too -- the only way to find out if the .asc exists is going to be to get it.  Maybe just don't get it if we're ignoring signatures?